docs/configuring.rst

```diff
@@ -146,3 +146,17 @@ The following snippet shows an example configuration for accessing the Bebop sys
 running on a login node, uses the ``SlurmProvider`` to interface with the scheduler, and uses the ``SrunLauncher`` to launch workers.
 
 .. literalinclude:: configs/bebop.py
+
+
+Pinning Workers to devices
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Many modern clusters provide multiple accelerators per compute note, yet many applications are best suited to using a
+single accelerator per task. funcX supports pinning each worker to difference accelerators using ``available_accelerators``
```

Comment:
- `difference` &rarr; `different`
- "... accelerators using THE ``available_accelerators`` ..."